_episodes/01-plone.md

```diff
@@ -3,48 +3,117 @@ title: Introduction to the Data Portal
+
+Inside the newest folder, there may be multiple files. If the titles/descriptions do not make it clear which one is newest you can always review the `last modified` date to determine this.
+
+When in doubt: please ask OTNDC Staff to help you find the newest file or export a clean file from the database for you. This will make everyone's lives easier!
+
+### Detection Extracts
+
+OTN and its partner Nodes create Detection Extracts on a semi-regular basis (approximately every 4 months) following a cross-Node coordinated detection matching event known as a Data Push. These Detection Extract files contain only the detections for the year corresponding to the suffix on the file name. See the detailed [detection extract documentation](https://members.oceantrack.org/data/otn-detection-extract-documentation-matched-to-animals) for more information.
+
+If you are ever interested in learning more about what a project has detected, you are welcome to check out the content in the Detection Extracts. Reminder that you **can not** share this information without reviewing the [OTN Data Policy](https://members.oceantrack.org/data/policies) or contacting OTNDC Staff for guidance.
+
+### Equipment
+
+This subfolder is the location where Shipping Lists can be posted and reviewed, if relevant. The shipping/inventory tracking workflow has moved to the [SnipeIT inventory system](https://ops.oceantrack.org/snipeit/).
+
+### Background
+
+This subfolder is the location where OTN Project Management staff will upload loan agreements, project plans and legal paperwork. This folder is only visible to OTN staff members by default.
+
+## FAQ page
+
+OTNDC has created a [Frequently Asked Questions](https://members.oceantrack.org/faq) page on the Data Portal. This will contain useful information about OTN's policies, Nodes, metadata, and data products. It is recommended you refer to this often and familiarize yourself with the answers to some of these questions, in case you are asked by partners.
+
+If there are topics you think should be added, please reach out to OTNDC - this page can be updated at any time to remain valuable to users.
+
+## Statistics Page
+
+OTNDC has created a [Statistics](https://members.oceantrack.org/statistics) page on the Data Portal. These statistics are updated every 4 months during a Data Push, and can be used for presentations, exploration and general interest. Summaries include:
+
+- an interactive barplot showing the number of tagged individuals, per species, coloured by IUCN status
+- a pie chart showing the total number of detections in the OTN database, by type
+- an interactive chart showing the number of receivers "active" each month, coloured by year
+- an interactive map showing the "currently active" receivers we know are still in the water at the time of publishing. Only Node's whose data policies allow the publishing of station locations are included (not FACT, ACT or MigraMar)
 
```

Comment:
we can mention that other stats visuals can be custom created through a request to jon etc?